src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -40,26 +40,28 @@ h2 {
   padding-bottom: 5px;
 }
 
-#types-list, #main-content {
-  position: absolute;
-  top: 0;
-  bottom: 0;
+body {
+  display: flex;
+}
+
+.sidebar, .main-content {
   overflow: auto;
 }
 
-#types-list {
-  left: 0;
-  width: 20%;
+.sidebar {
+  width: 30em;
```

Comment:
This is unacceptable on vertical monitors.

Reply:
What exactly is unacceptable?

Reply:
That it takes up more than a third of my monitor.

Reply:
Which browser and resolution?

Reply:
True, on my MBA screen (1440×900) looks unnaturally wide too... :(